src/libraries/Native/Unix/System.Globalization.Native/pal_calendarData.c

```diff
@@ -5,12 +5,22 @@
 #include <assert.h>
 #include <stdlib.h>
 #include <string.h>
-#include <strings.h>
-
 #include "pal_locale_internal.h"
 #include "pal_errors_internal.h"
 #include "pal_calendarData.h"
 
+#if defined(TARGET_UNIX)
+#include <strings.h>
+
+#define STRING_COPY(localeNamePtr, size, locale, capacity) \
```

Comment:
Kind of a nit, but if you're going to roll-your-own string copy macro, consider changing the names of parameters to be more generic, perhaps something like:
`STRING_COPY(destination, destinationSize, source, count)`

Reply:
Good idea.